tests/python/gaia-ui-tests/gaiatest/apps/clock/regions/stopwatch.py

```diff
@@ -0,0 +1,63 @@
+ # This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import expected, By, Wait
+
+from gaiatest.apps.clock.app import Clock
+from gaiatest.apps.base import PageRegion
+
+
+
+class StopWatch(Clock):
+    _stopwatch_view_locator = (By.CSS_SELECTOR,'#stopwatch-panel.active')
```

Comment:
I don't think you need the .active part here. This locator is used to make sure this panel is visible anyway.